docs/guides/local-db.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: "Local Database Setup"
+description: "Set up PostgreSQL for Suga local development."
+---
+
+Suga doesn’t auto-provision databases locally, giving you full choice and control. Here’s how to set up PostgreSQL for local use and any provider (Neon, Supabase, RDS) for production.
+
+<Note>
+  **Before you begin:** Make sure you have Suga CLI installed. See our [Installation Guide](/cli/installation) for platform-specific instructions.
+</Note>
+
+<AccordionGroup>
+  <Accordion title="Step 1: Initialize Project" icon="folder-plus">
+    **Start with a fresh Suga project**
+
+    First, login to Suga if you haven't already:
+
+    ```bash
+    suga login
+    ```
+
+    Create a new project and select the Go template:
+
+    ```bash
+    suga new my-app
+    ```
+
+    When prompted, select the Go template (go-starter).
+
+    Enter your project:
+
+    ```bash
+    cd my-app
+    ```
+
+    Generate the Suga SDK to enable access to cloud resources:
+
+    ```bash
+    suga generate --go --go-out ./suga --go-package-name suga
+    go mod tidy
+    ```
+  </Accordion>
+  <Accordion title="Step 2: Setup PostgresSQL with a Docker container" icon="docker">
+    <Tip>
+      Already have PostgreSQL running? Skip this step and use your existing database.
+    </Tip>
```

Comment:
I'm confused by this section and tip, isn't this the intention of the guide?

Step 1 and Step 3 are both different guides (how to setup a project, adding a database to a project). I think they should both be moved/removed.

This page should assume I have a project and I'm trying to do what the title say "Local Database Setup".

Reply:
Makes sense, I've refocussed it on just setting up the DB removing the unnecessary.